src/content/app/genome-browser/state/track-settings/trackSettingsSlice.ts

```diff
@@ -247,6 +271,20 @@ const trackSettingsSlice = createSlice({
 
       trackSettingsState.settings.showTranscriptIds = shouldShowTranscriptIds;
     },
+    addSettingsForTrack(
+      state,
+      action: PayloadAction<{
+        genomeId: string;
+        trackId: string;
+        trackSettings: TrackSettings;
+      }>
+    ) {
+      const { genomeId, trackId, trackSettings } = action.payload;
+      if (!state[genomeId]) {
+        state[genomeId] = { settingsForIndividualTracks: {} };
```

Comment:
Why do you need a nesting with `settingsForIndividualTracks` Can't we have all settings inside `state.browser.trackSettings`?

Reply:
I was asking myself the same question, trying to remember why we had it. I think this may be a remnant from the time when we had the all tracks/this track toggle, which we stored in redux, and which referred to all tracks in a genome. We aren't doing so now; so if we are feeling brave, we can remove this key. But in a subsequent PR, not in this one.

Reply:
ok. Yeah better remove it if this is unnecessary. Please could you create a ticket? Thanks